app/common/prescription/views/prescription-arv.html

```diff
@@ -76,7 +76,7 @@ <h3 class="panel-title">{{'CLINIC_PRESCRIPTION_CURRENT_REGIMEN'| translate}}</h3
 
             <!-- ARV Plan elements -->
             <div class="form-group form-group-lg"
-                         ng-class="{'has-error': prescriptionForm.artPlan.$invalid && vm.showMessages}">
+                         ng-class="{'has-error': prescriptionForm.artPlan.$invalid && vm.showMessages}" ng-init="vm.initArvPlans()">
```

Comment:
I would prefer that ```initArvPlans``` is called inside the controllers ```activate``` function. [syleguide](https://github.com/johnpapa/angular-styleguide/tree/master/a1#style-y080).